src/UI/Implementation/Render/ilJavaScriptBinding.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/* Copyright (c) 2016 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Implementation\Render;
+
+/**
+ * Wraps global ilTemplate to provide JavaScriptBinding. 
+ */
+class ilJavaScriptBinding implements JavaScriptBinding {
+	/**
+	 * @var \ilTemplate
+	 */
+	private	$global_tpl;
+
+	/**
+	 * @var	integer
+	 */
+	private $id_count = 0;
```

Comment:
It may be that the question arises due to an improper naming of `JavaScriptBinding`. This should be the object which knows how to actually bind javascript to HTML, which to me seems to be unique in the system at a given point in time. So it might be i don't understand the question, but the answer to "two binders in parallel?" is "no".

I'll still randomize the ids and prefix them as well.